src/coldcard.test.ts

```diff
@@ -524,7 +525,7 @@ describe("ColdcardSignMultisigTransaction", () => {
         outputs: transactions[0].outputs,
         bip32Paths: transactions[0].bip32Paths,
       });
-      const result = interaction.request().toBase64();
+      const result = interaction.request().data.toBase64();
```

Comment:
wow was this just broken?

Reply:
It was broken only due to a change in unchained-bitcoin v1.0.0. I think the psbt is on both the request root and `data`, but it was only typed at the `data` attribute. So, it worked the way it was without typescript. @nk1tz would have more info if he remembers. I think he contributed the typing change on unchained-bitcoin.